_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw_utils.cpp

```diff
@@ -3461,7 +3462,7 @@ void ConfigureTask(
     {
         for (mfxU16 i = 0; i < parRoi->NumROI; i ++)
         {
-            memcpy_s(&task.m_roi[i], sizeof(RoiData), &parRoi->ROI[i], sizeof(RoiData));
+        	task.m_roi[i] = parRoi->ROI[i];
```

Comment:
see https://github.com/Intel-Media-SDK/MediaSDK/pull/724 to get the idea of how to do this without custom assignment operator